doc/jointcal.tex

```diff
@@ -29,527 +43,603 @@
-the relative astrometry between images is considerably more precise
-than the accuracy of external catalogs, where ``more precise'' can be as large
-as two orders of magnitude. For applications where the
+
+With deep astronomical images, it is extremely common that the relative
+astrometry and photometry between images is considerably more precise
+than the accuracy of external catalogs, where ``more precise'' can
+be as large as two orders of magnitude. For applications where the
 quality of relative astrometry is important or vital, it is important
 to rely on some sort of simultaneous astrometry solution, if possible
-optimal in a statistical sense. 
-
-This package performs a least-square fit to a set of images.
-Since it aims at statistical optimality, we maximize 
-the likelihood of the measurements with respect to all 
-unknown parameters required to describe the data.
-These parameters consists mostly in two sets: the position
-(on the sky) of the objects in common, and the mapping
-of each image to the sky. To these obvious parameters,
-one can add proper motions (where applicable), and 
-parameters describing the differential effect of atmospheric refraction
-on the position of objects. It is clear that one cannot
-fit simultaneously the position on the sky and the mappings
-from CCD coordinates to the sky, without extra constraints:
-the ``sky'' coordinate system is then undefined, and one needs
-reference positions in order to fully define this frame.
-So far, we have used the USNO (A 2.0) catalog for this purpose.
-
-
-SCAMP\footnote{see \href{http://www.astromatic.net/software/scamp}{http://www.astromatic.net/software/scamp}} is the reference package for simultaneous astrometry in
-astronomy, at least for relative alignment of wide-field images prior
-to stacking. Regarding optimization, SCAMP follows a somehow different
-route from ours: it does not optimize over the position of common
-objects but rather minimizes the distance between pairs of transformed
-measurements of the same object. This approach is not a maximum
-likelihood optimization, and hence is likely sub-optimal statistics
-wise. We do not know how sub-optimal it is, but the main drawback of
-SCAMP in the context of LSST is the fact that it is a program and not
-a library, and hence not flexible regarding formats of images and
-catalogs.  But since SCAMP has been used for almost a decade in
-production by various teams, the quality checking tools it provides
-should likely be reproduced in the context of our package. We provide
-residual ntuples and hope that the first serious users will contribute
-plotting tools.
+optimal in a statistical sense.
+
+This package performs a least-square fit to a set of images. Since
+it aims at statistical optimality, we maximize the likelihood of the
+measurements with respect to all unknown parameters required to describe
+the data. These parameters consists mostly in two sets: the position
```

Comment:
Suggest: These parameters are mostly in two sets: the position on the sky of the objects in common between images, and...